requirements/base.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
```

Comment:
**Note to reviewer:** Although there are some minor upgrade changes, the only real change was adding ddt for testing.
[Mark resolved when done reading.]